### ISSUE TITLE
Record in temp location was indexed with permanent location

### ISSUE THREAD

**Issue by christinach** [pulibrary/bibdata#2428]
### Expected behavior
Records in temporary locations besides `RES_SHARE$IN_RS_REQ`RES_SHARE$IN_RS_REQ` should be indexed with the temporary locaiton in holdings1_display

### Actual behavior
Records in temporary location: stokes$spir were indexed with the permanent location
```
<datafield tag="952" ind1=" " ind2=" ">
      <subfield code="a">2021-07-12 20:23:46</subfield>
      <subfield code="8">22567229310006421</subfield>
      <subfield code="b">Firestone Library</subfield>
      <subfield code="c">stacks: Firestone Library</subfield>
      <subfield code="e">false</subfield>
    </datafield>
    <datafield tag="876" ind1=" " ind2=" ">
      <subfield code="0">22567229310006421</subfield>
      <subfield code="a">23567229300006421</subfield>
      <subfield code="j">1</subfield>
      <subfield code="z">spir</subfield>
      <subfield code="d">2004-05-10 20:00:00 US/Eastern</subfield>
      <subfield code="p">32101049383092</subfield>
      <subfield code="t">1</subfield>
      <subfield code="y">stokes</subfield>
    </datafield>
```

### Impact of this bug
Users interested in the specific record think the record is in Firestone$stacks

### Implementation notes
This was reported by [Tracy H. in the catalog channel](https://pulibrary.slack.com/archives/CA0UY0Q07/p1722011177444009).

mms_id: 9941616013506421
mms_id: 99129005733006421

**Comment by christinach** [comment]
@mzelesky and I reviewed the record history in Alma and the record was changed on: 
7/25 at 10:22am eastern time
7/26 at 12:14pm eastern time
This means that the record should have been published twice, once on 7/25th after 10:30am and once on the 7/26th after 12:15pm

@mzelesky looked through all the published files in alma and the only time we found the record was published was on 7/24th after 1pm. On 7/24th the record was still in the permanent location this is why it was indexed in Firastone$stacks. 

The marc record in the main ticket comment was manually published by @christinach on 7/25th and it had the temporary location changes from alma.
